osmaxx/api_client/API_client.py

```diff
@@ -15,7 +15,6 @@ class RESTApiClient:
 
     def __init__(self, service_base):
         self.service_base = service_base
-        self.client = requests.session()
```

Comment:
I realized that, and AFAIK that line stems from an idea where I implemented it this way - but forgot about it again, and didn't remember while reviewing.

I created the issue #508 for this, since this doesn't belong in here.